src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -155,6 +161,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     this._eventManager = new EventManager();
     this._maybeSetDrmPlayback();
     this._progressiveSources = config.sources.progressive;
+    this._useExternalCaptions = config.playback && config.playback.useNativeTextTrack;
```

Comment:
why to save this value on `this`?
need to clean in change media?

Reply:
using it once, so moved it there, and didn't store it on `this`